src/web/components/ArticleHeadline.tsx

```diff
@@ -161,6 +161,26 @@ export const ArticleHeadline = ({
     tags,
     byline,
 }: Props) => {
+    const isPrintShop = tags.find(
+        tag => tag.id === 'artanddesign/series/guardian-print-shop',
```

Comment:
is `artanddesign/series` important?

Reply:
The string `artanddesign/series/guardian-print-shop` is the literal id for the tag so I think it's correct to use it all.